code/src/layouts/app-products/AppProductsList.tsx

```diff
@@ -0,0 +1,47 @@
+import Box from "@mui/material/Box";
+
+import { useGetProductsQuery } from "@/store/api/productsApi";
+import PageWrapper from "@/layouts/app-wrapper/PageWrapper";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+import { Product } from "@/types/product.types";
+
+import "@/layouts/app-products/AppProductsList.scss";
+
+const AppProductsList = () => {
+  const { data: products } = useGetProductsQuery();
+
+  const renderProducts = (products: Product[]) => {
+    return products.map((product) => (
+      <Box className="spa-productsList__productContainer" key={product.id}>
```

Comment:
I suggest to create a separate component for current item and just pass there `product` as a prop

Reply:
I will creale aditional issue for Product Cart, because we should discuss wich additional props we need to make it reusable as much as possible

Reply:
> I will creale aditional issue for Product Cart, because we should discuss wich additional props we need to make it reusable as much as possible

It's just refactor your existing code.
You can create separate component for product item and that's all